.github/workflows/flake8.yml

```diff
@@ -0,0 +1,66 @@
+name: Python code quality check
+
+# Controls when the action will run.
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  # This workflow contains two job called "flake8-grass7-scripts" and "flake8-gui-wxgui-scripts"
+
+  flake8-grass7-scripts:
+    runs-on: ubuntu-20.04
```

Comment:
Here `name:` would probably do a better job of telling which piece of the matrix is running in the Checks overview. See e.g. gcc.yml in core. The current automatic one is e.g. _flake8-grass7-scripts (grass7/general)_ but it could be just `Flake8 ${{ matrix.directory }}` (so e.g. _Flake8 grass7/general_).

Reply:
fixed